drivers/i2c/Kconfig.nrfx

```diff
@@ -56,4 +56,14 @@ config I2C_3_NRF_TWIM
 	  This peripheral accepts transfers from RAM only,
 	  if provided buffer is placed in flash, transfer will fail.
 
+config I2C_NRF_WORKAROUND_ERRATA89
+	bool "Workaround errata 89"
+	depends on SOC_NRF52832
+	help
+	  When GPIOTE (gpio interrupts) is active and SPI or TWI is
+	  also active, there is excessive current drain during low
+	  power (sleep/idle).
+	  This implements the workaround recommended in errata 89,
+	  which toggles the power to the TWI or SPI module after suspend.
```

Comment:
Anomaly 89 in nRF52832 is related to EasyDMA, so it concerns TWIM and SPIM, not TWI and SPI, which are a bit different peripherals.

Reply:
Just to note,  although the errata does say it only applies to TWIM/SPIM, I had increased current drain regardless of whether SPI/TWI or SPIM/TWIM was used.